src/components/VPopover/VPopover.vue

```diff
@@ -104,6 +104,10 @@ export default defineComponent({
      * @default undefined
      */
     labelledBy: { type: String },
+    /**
+     * the z-index to apply to the popover content
+     */
+    zPos: { type: Number },
```

Comment:
Should we set a default value?
I am afraid the z-indices already need a good review :(

Reply:
The issue was that a lot of absolutely positioned things were coming over the popup, notably the related images.